### ISSUE TITLE
Some question about the threat model

### ISSUE THREAD

**Issue by HuKeping** [docker/notary#832]
Hi there,

As the [docs](https://github.com/docker/notary/blob/master/docs/service_architecture.md#possible-attacks-given-the-credentials-compromised) about `Threat model` says,  if the delegation key compromised, the attacker can malicious/remove content on the particular delegation path. 

But what does the `Notary Service write-capable credentials` means?  Does it mean the JWT authentication mechanism that every request to the notary-server will ask for a access credential first?

It would be great if there would be a little example to illustrate that.

**Comment by cyli** [comment]
Hi @HuKeping!  Yes it's the JWT auth mechanism.

And do you mean just an example of the auth mechanism at work generally?  Or something more specific?

**Comment by HuKeping** [comment]
Oh never mind, I was just not sure it was the JWT auth mechanism and was thinking what could it be because there is not an example to help to explain "why the attacker have already get the decrypted key but still can not talk to notary-server".  But since it is the JWT, there is no doubt then. Thank you @cyli

**Comment by HuKeping** [comment]
Hi @cyli ! Why the docs said "An attacker can add malicious content, remove legitimate content from a collection, and mix up the targets in a collection" in the term that the snapshot keys has not been rotated to the service side yet? See [docs about thread model](https://github.com/docker/notary/blob/master/docs/service_architecture.md#possible-attacks-given-the-credentials-compromised)

**Comment by cyli** [comment]
You're right it can be confusing since mixing is can be used to mean taking
different versions of different metadata files.  It was added because the
delegation key make arbitrary changes to the delegation roles under the
role they're signed into.  By "mixing" we mean you can basically reproduce
a different version of an underlying delegation (just copy all the data
over) - essentially just adding and removing targets.

Do you think we should take it out since "adding" and "removing" targets
basically covers that last case?

**Comment by HuKeping** [comment]
I'm fine with that, just tried to get it clear.

Thanks @cyli

**Comment by riyazdf** [comment]
@HuKeping I'll close this issue for now then, but please let us know if you have more questions!  And if you think we need better explanation for certain topics in the docs, we can put them in #828 or a similar issue
